packages/fast-color-explorer/app/gradient.tsx

```diff
@@ -19,9 +20,9 @@ const styles: any = {
             height: "6px",
             margin: "0 auto",
             content: "''",
-            opacity: "0.5",
+            opacity: "0.7",
             position: "relative",
-            border: "solid 1px white",
+            border: "solid 1px currentColor",
```

Comment:
currentColor should be currentcolor, see: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentColor_keyword